department-of-veterans-affairs/abd-vro
# Convert all application properties to yaml

## What was the problem?

The use of `application.properties` files was not being loaded correctly into Spring applications for `svc-lighthouse-api` and `svc-bip-api`.

Associated tickets or Slack threads:
- #2180 

## How does this fix it?[^1]

Converts all the `*.properties` files to use yaml format which seems to actually be loaded. 

## How to test this PR
- Deployed to `dev` and shown to resolve the issues with `rabbitmq` auth credentials


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
